.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  build:
+    uses: ./.github/workflows/publish.yml
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - ["3.11", cp311]
+          - ["3.12", cp312]
+          - ["3.13", cp313]
+        os_arch:
+          - [ubuntu-latest, manylinux_x86_64]
+          - [windows-latest, win_amd64]
+          - [macos-13, macosx_x86_64] # macos-13 is the last x86-64 runner
```

Comment:
Not sure about this?
```macos-latest-large, macos-14-large, macos-15-large``` are all `x86-64` runners.

Ref.
https://github.com/actions/runner-images/blob/main/README.md#available-images

Reply:
ahh. it was simply copied from xtgeos github actions :sweat_smile: